lib/between_meals/knife.rb

```diff
@@ -80,6 +93,20 @@ def cookbook_upload(cookbooks)
       end
     end
 
+    def berks_cookbook_upload(cookbooks)
+      # cookbooks: array
+      # cookbook_paths: array
+      if cookbooks.any?
+        @cookbook_dirs.each do |path|
+          cookbooks.each do |cb|
+            @logger.warn("Running berkshelf on cookbook: #{cb}")
```

Comment:
I would just check to see if the directory exists

`next unless File.exists?("#{path}/#{cb}")`

Seem reasonable?